src/config/graphical-session/gnome.md

```diff
@@ -2,27 +2,28 @@
 
 ## Pre-installation
 
+GNOME supports both Xorg and Wayland sessions. Follow [Wayland](./wayland.md),
+[Xorg](./xorg.md) sections to setup your preferred environment.
+
 Install the `dbus` package, ensure the `dbus` service is enabled, and reboot for
 the changes to take effect.
 
 ## Installation
 
-Install the `gnome` package for a GNOME environment which includes GNOME
-applications.
+Install the `gnome` package for a GNOME environment which includes the base
+GNOME desktop and a subset of GNOME applications. Additional applications are
+available via the `gnome-apps` package.
+
+A minimal GNOME environment can be created by installing the `gnome-core`
+package. Note, however, that not all GNOME features may be present or
+functional.
+
+If you require [ZeroConf](http://www.zeroconf.org/) support, install the `avahi`
+package and enable the `avahi-daemon` service.
 
-A minimal GNOME environment can be created by installing the `mesa-dri`,
-`gnome-session`, `gdm` and `adwaita-icon-theme` packages. (Note, however, that
-not all GNOME features may be present or functional.)
+## Starting GNOME
 
 The `gdm` package provides the `gdm` service for the GNOME Display Manager;
 [test the service](../services/index.md#testing-services) before enabling it.
 GDM defaults to providing a Wayland session via the `mutter` window manager, but
-an X session can be chosen instead.
-
-If you wish to start an X-based GNOME session from the console, use
-[startx](./xorg.md#startx) to run `gnome-session`.
-
-GNOME applications can be installed via the `gnome-apps` package.
-
-If you require [ZeroConf](http://www.zeroconf.org/) support, install the `avahi`
-package and enable the `avahi-daemon` service.
+an Xorg session can be chosen instead.
```

Comment:
I tried making this change once, but using `X` is the correct name, since `Xorg` is "only" an implementation.

Reply:
Reverted. I was trying to make it consistent with "Gnome on Xorg" option of GDM.